src/reactapp/build/i18n/de_DE.json

```diff
@@ -0,0 +1,66 @@
+{
+    "%1 is required": "%1 ist erforderlich",
+    "Actions": "Aktionen",
+    "Address switching failed, sorry.": "Adressumschaltung fehlgeschlagen, sorry.",
+    "Billing Information": "Rechnungsinformationen",
+    "Billing address update failed. Please try again": "Aktualisierung der Rechnungsadresse fehlgeschlagen. Bitte erneut versuchen",
+    "Billing address updated successfully": "Rechnungsadresse erfolgreich aktualisiert",
+    "Billing address updated successfully.": "Rechnungsadresse erfolgreich aktualisiert.",
+    "Cart updated successfully": "Warenkorb erfolgreich aktualisiert.",
+    "City": "Stadt",
+    "Company": "Firma",
+    "Country": "Land",
+    "Create a new address": "Neue Adresse erstellen",
+    "E-mail": "E-Mail",
+    "EDIT": "BEARBEITEN",
+    "Email": "E-Mail",
+    "Email is invalid": "E-Mail ist ungültig",
+    "Email is required": "E-Mail ist erforderlich",
+    "Email is successfully attached to your cart.": "E-Mail wurde erfolgreich an Ihren Warenkorb angehängt",
+    "Firstname": "Vorname",
+    "I will sign-in and checkout": "Ich melde mich an und checke aus",
+    "Item": "Artikel",
+    "Lastname": "Nachname",
+    "My billing & shipping address are same": "Meine Rechnungs und Lieferadresse sind identisch",
+    "No cart items available": "Keine Warenkorbartikel verfügbar",
+    "No payment methods available at the moment": "Derzeit sind keine Zahlungsmethoden verfügbar",
+    "No shipping methods available at the moment": "Derzeit sind keine Versandmethoden verfügbar",
+    "No, I like to continue as guest": "Nein, ich möchte als Gast fortfahren",
+    "OR": "ODER",
+    "Order Summary": "Bestellübersicht",
+    "Password": "Passwort",
+    "Password is required": "Passwort ist erforderlich",
+    "Payment Methods": "Zahlungsmethoden",
+    "Payment method added successfully.": "Zahlungsmethode erfolgreich hinzugefügt.",
+    "Phone": "Telefon",
+    "Place Order": "Bestellung aufgeben",
+    "Postal Code": "Postleitzahl",
+    "Price": "Preis",
+    "Price: {}": "Preis: {}",
+    "Product Details": "Produktdetails",
+    "Qty": "Menge",
+    "Required": "Erforderlich",
+    "Shipping Information": "Versandinformationen",
+    "Shipping Methods": "Versandmethoden",
+    "Shipping address update failed. Please try again": "Aktualisierung der Versandadresse fehlgeschlagen. Bitte erneut versuchen",
+    "Shipping address updated successfully": "Versandanzeige Kleid erfolgreich aktualisiert",
```

Comment:
correct translation: "Versandadresse erfolgreich aktualisiert" (as below)

Reply:
(once with `.` and once without)

Reply:
This is corrected 👍